### ISSUE TITLE
Folder does not load when a file with a percentage (%) sign is in folder

### ISSUE THREAD

**Issue by sebasm12** [nextcloud/server#11892]
Nextcloud version: 14.0.3
Operating system and version: CENTOS 7.5.1804 (Core)
Apache or nginx version: lighttpd 1.4.50
PHP version: 7.0.32

The issue you are facing:
When a file is present in a folder with a percentage sign in the file name (e.g. "TestFile10%.png"), then the folder does not open in the web interface, and the loading animation (rotating circle) keeps spinning.

Is this the first time you've seen this error?: Yes

Steps to replicate it:

1. Go do some subfolder
2. Create a textfile with filename "Test10%.txt
3. Go to root
4. Try to navigate to the folder again -> The folder does not load

The output of your Nextcloud log in **Admin > Logging**:
```
Nothing to show
```

The output of your config.php file in `/path/to/nextcloud` (make sure you remove any identifiable information!):
```
<?php
$CONFIG = array (
  'updatechecker' => false,
  'instanceid' => 'XXX',
  'passwordsalt' => 'XXX',
  'secret' => 'XXX',
  'updater.secret' => 'XXX',
  'trusted_domains' =>
  array (
    0 => 'XXX.XXX.com',
  ),
  'datadirectory' => '/home/XXX',
  'overwrite.cli.url' => 'https://XXX.XXX.com',
  'dbtype' => 'mysql',
  'version' => '14.0.3.0',
  'dbname' => 'XXX',
  'dbhost' => 'localhost',
  'dbtableprefix' => 'XXX',
  'dbuser' => 'XXX',
  'dbpassword' => 'XXX',
  'logtimezone' => 'UTC',
  'installed' => true,
  'memcache.local' => '\\OC\\Memcache\\APCu',
  'maintenance' => false,
  'theme' => '',
  'loglevel' => 2,
  'singleuser' => false,
  'mail_smtpmode' => 'smtp',
  'mail_smtphost' => 'smtp.gmail.com',
  'mail_smtpport' => '465',
  'mail_smtpauthtype' => 'LOGIN',
  'mail_smtpauth' => 1,
  'mail_from_address' => 'XXX',
  'mail_domain' => 'XXX',
  'mail_smtpsecure' => 'ssl',
  'mail_smtpname' => 'XXX',
  'mail_smtppassword' => 'XXX',
);

```

**Comment by sebasm12** [comment]
Duplicate of #11405

**Comment by MorrisJobke** [comment]
Fixed in #11744 and will be shipped in 14.0.4

**Comment by MorrisJobke** [comment]
Duplicate of #11695
